Update [analytics] section to include the ability to set to false (#71473)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

## [analytics]

### enabled

> Also known as "Usage Analytics". When set to false it will disable the writers that read/write from/to Grafana databases. Default

Reviewer: ```suggestion
This option is also known as _usage analytics_. When `false`, this option disables the writers that read/write from and to Grafana databases. Default
```

> This option is also known as _usage analytics_. When `false`, this option disables the writers that read/write from and to Grafana databases. Default

Reviewer: ```suggestion
This option is also known as _usage analytics_. When `false`, this option disables the writers that read/write from and to the Grafana databases. The default
```
